views-thymeleaf/src/main/java/io/micronaut/views/thymeleaf/LinkBuilder.java

```diff
@@ -32,6 +34,16 @@
 @Singleton
 public class LinkBuilder extends StandardLinkBuilder {
 
+    @Nullable
+    private String contextPath;
+
+    @Inject
+    public LinkBuilder(@Nullable @Property(name = "micronaut.server.context-path") String contextPath) {
+        this.contextPath = contextPath;
+    }
+
+    public LinkBuilder() { }
```

Comment:
annotate old one with @Depreacated and tell to use the other controller

Reply:
resolved via https://github.com/micronaut-projects/micronaut-views/pull/348/commits/08005a42c634addc476418d105594c618a60ac11